cmd/webhook.go

```diff
@@ -96,7 +96,7 @@ var webhookCmd = &cobra.Command{
 			Scheme:                 scheme,
 			MetricsBindAddress:     metricsAddr,
 			HealthProbeBindAddress: healthzAddr,
-			Port:                   9443,
+			Port:                   443,
```

Comment:
I would make this configurable to CLI flags to avoid port collisions when a user wants to run this with `hostNetwork=true`.